pkgs/top-level/python-packages.nix

```diff
@@ -6058,6 +6058,37 @@ let
     };
   };
 
+  django_silk = makeOverridable ({ django ? self.django }: buildPythonPackage rec {
```

Comment:
But every other `django-something` package in nixpkgs is named `django_something` - that's what I was following.